metabase/metabase
# Allow clearing dashboard description

Closes https://github.com/metabase/metabase/issues/32346

### Description

I believe this might be a regression since https://github.com/metabase/metabase/pull/23608 was introduced (I haven't tested commits before this PR). Since `isOptional` was introduced shorted before that PR https://github.com/metabase/metabase/pull/23532.


### How to verify

1. Create a dashboard if not exist
1. Go to a dashboard
1. Add description
1. Try to clear description, the description should be cleared

### Demo

https://www.loom.com/share/c493db1ffdd64912acb7efc5792a7998

### Checklist
